server/chat-plugins/chat-monitor.ts

```diff
@@ -667,8 +667,65 @@ export const commands: ChatCommands = {
 		}
 		this.globalModlog(`ALLOWNAME`, target);
 	},
+	testfilter(target, room, user, connection, cmd) {
+		this.privatelyCheckCan('lock');
+		this.checkBroadcast(true);
+		if (room && ['staff', 'upperstaff'].includes(room.roomid)) this.runBroadcast();
```

Comment:
Why checkBroadcast and then runBroadcast if it's in Staff? Can't you just `if (room && ['staff', 'upperstaff'].includes(room.roomid)) this.runBroadcast(true);`?

Reply:
bc i am dumb and forgot to remove the line